### PR TITLE
[JBEAP-JBEAP-5239] - store dependency versions in properties

### PR DESCRIPTION
**Pull request by psakar**

### CODE REVIEW

File: src/it/bom-with-version-properties/verify.groovy [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+ File expectedFile = new File(basedir, "expected/pom.xml")
+
+ String fileContents = file.getText('UTF-8')
+ String expectedFileContents = expectedFile.getText('UTF-8')
+
+ def isDifferent = !fileContents.equals(expectedFileContents)
+ if (isDifferent) {
+    System.err.println("Generated " + file.absolutePath + " differs from expected " + expectedFile.absolutePath)
+    return false
+ }
```
Comment (pgier) [inline src/it/bom-with-version-properties/verify.groovy]: Some of the files seem to be missing newlines at the end.

File: src/main/java/org/jboss/maven/plugins/bombuilder/BuildBomMojo.java [hunk 2]
```diff
@@ -69,7 +69,7 @@
      * BOM output file
      */
     @Parameter( defaultValue = "bom-pom.xml" )
-    private String outputFilename;
+    String outputFilename;
```
Comment (pgier) [inline src/main/java/org/jboss/maven/plugins/bombuilder/BuildBomMojo.java:82]: Should this use getters and setters instead?

**Review by pgier — APPROVED** [review]
Looks good to me, I just added some minor comments

File: src/it/bom-with-version-properties/verify.groovy (hunk 1 above)
Comment (psakar) [reply src/it/bom-with-version-properties/verify.groovy]: added new lines - please see the latest commit of this PR

File: src/main/java/org/jboss/maven/plugins/bombuilder/BuildBomMojo.java (hunk 2 above)
Comment (psakar) [reply src/main/java/org/jboss/maven/plugins/bombuilder/BuildBomMojo.java:82]: I did not want to make the outputFilename publicly available or make it property. I just need to set the value in test, hence default visibility. If you prefer setter, I can change it

Comment (pgier) [reply src/main/java/org/jboss/maven/plugins/bombuilder/BuildBomMojo.java:82]: I think it's fine without the setter.

**Pull request merged**